windows/scripts/get_engine_artifacts.bat

```diff
@@ -0,0 +1,15 @@
+:: Copyright 2018 Google LLC
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http://www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+@echo off
+xcopy /y /d /q ..\..\..\flutter\bin\cache\artifacts\engine\windows-x64\icudtl.dat ..\dependencies\engine
```

Comment:
Please follow the structure used in update_flutter_engine, separating the Flutter tree path from the relative portion even though the tree path is currently hard-coded, with a TODO on the former (which can now reference bug #88)